TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -295,7 +295,7 @@ private struct BufferItem {
                 (queueItem_.vehicleType & ExtVehicleType.RoadVehicle) != ExtVehicleType.None;
 
             isLaneArrowObeyingEntity_ =
-                (!Options.relaxedBusses || queueItem_.vehicleType != ExtVehicleType.Bus) &&
+                (!Options.relaxedBusses || (queueItem_.vehicleType != ExtVehicleType.Bus && queueItem_.vehicleType != ExtVehicleType.Tram)) &&
                 (vehicleTypes_ & LaneArrowManager.VEHICLE_TYPES) != VehicleInfo.VehicleType.None &&
```

Comment:
this already checks for that:
`queueItem_.vehicleType & LaneArrowManager.EXT_VEHICLE_TYPES) != ExtVehicleType.None`

Reply:
You are right - The problem is somewhere in the pathfinding related to the nodes not beeing connected by the LaneConnectors...